modules/rds/aurora-db/main.tf

```diff
@@ -16,7 +17,7 @@ resource "aws_rds_cluster" "aurora-cluster" {
     final_snapshot_identifier     = "${var.name}-aurora-cluster"
     skip_final_snapshot           = "${var.skip_final_snapshot}"
     vpc_security_group_ids        = [
-        "${aws_security_group.aurora_db.id}"
+        "${aws_security_group.aurora_db_vpc.id}"
```

Comment:
you are always referencing aurora_db_vpc so, wondering how will it work? I mean I want to use aurora_db_sg!

Reply:
done